dao-services/src/main/java/org/exoplatform/wiki/migration/AttachmentsMigration.java

```diff
@@ -0,0 +1,315 @@
+package org.exoplatform.wiki.migration;
+
+import liquibase.change.custom.CustomTaskChange;
+import liquibase.database.Database;
+import liquibase.database.jvm.JdbcConnection;
+import liquibase.exception.CustomChangeException;
+import liquibase.exception.DatabaseException;
+import liquibase.exception.SetupException;
+import liquibase.exception.ValidationErrors;
+import liquibase.resource.ResourceAccessor;
+import org.exoplatform.commons.file.model.FileItem;
+import org.exoplatform.commons.file.services.FileService;
+import org.exoplatform.commons.file.services.NameSpaceService;
+import org.exoplatform.commons.utils.CommonsUtils;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.wiki.jpa.JPADataStorage;
+
+import java.io.ByteArrayInputStream;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Calendar;
+
+/**
+ * Attachment Migration service Created by The eXo Platform SAS Author :
+ * eXoPlatform@exo@exoplatform.com
```

Comment:
Please add a more meaningful comment.
We should explain that this is the implementation of a custom tag in liquibase.
And we should remove this "Created by ...", it pollutes.